Wikipedia/Code/ReadingThemesControlsViewController.swift

```diff
@@ -1,15 +1,17 @@
 import UIKit
 
-@objc public protocol WMFReadingThemesControlsViewControllerDelegate {
+public protocol WMFReadingThemesControlsViewControllerDelegate: class {
     
     func fontSizeSliderValueChangedInController(_ controller: ReadingThemesControlsViewController, value: Int)
+    func toggleSyntaxHighlighting(_ controller: ReadingThemesControlsViewController)
 }
 
 @objc(WMFReadingThemesControlsViewController)
 open class ReadingThemesControlsViewController: UIViewController {
```

Comment:
should this be `open` or would `public` suffice?

Reply:
@nambatee good point - actually I don't even think public is necessary since it's not a part of the WMF framework...I have removed, let me know what you think.

Reply:
awesome! 🙌